web/tests/example8.test.js

```diff
@@ -82,7 +82,11 @@ run_test("typing_events.render_notifications_for_narrow", ({override, mock_templ
     const group = [anna.user_id, vronsky.user_id, levin.user_id, kitty.user_id];
     const conversation_key = typing_data.get_direct_message_conversation_key(group);
     const group_emails = `${anna.email},${vronsky.email},${levin.email},${kitty.email}`;
-    narrow_state.set_current_filter(new Filter([{operator: "dm", operand: group_emails}]));
+    message_lists.current = {
+        data: {
+            filter: new Filter([{operator: "dm", operand: group_emails}]),
+        },
+    };
```

Comment:
Could this easily use `new MessageList/MessageListData` too, rather than a mock?

Reply:
No, we will have to mock other templates and methods.